app/views/budgets/investments/_investment_show.html.erb

```diff
@@ -137,6 +137,15 @@
           <span class="icon-budget"></span>
         </div>
         <br>
+	<% if investment.unfeasible? %>
```

Comment:
Please can you fix the indentation of this file (using two spaces) 🙏 😉 

```
<% if investment.unfeasible? %>
  <div class="callout warning">
    <%= t("budgets.investments.show.project_unfeasible_html") %>
  </div>
<% elsif !investment.selected? %>
  <div class="callout warning">
    <%= t("budgets.investments.show.project_not_selected_html") %>
  </div>
<% end %>
```

Thanks!

Reply:
I have no idea how that happened, I thought I had my emacs already configured for that. Anyway thanks for letting me know. I will have a look into that. I hope now it's OK

Reply:
Don't worry! now looks perfect! 😌